### ISSUE TITLE
GV-P: Expand coverage of distant supervision

### ISSUE THREAD

**Issue by ajratner** [ajratner/dd-genomics#103]
@amwenger @Colossus @gbgbg Moving this to separate issue from #89.  Currently we link HGVS nomenclature to HPO terms and seem to get low coverage- can we e.g. get other supervision sources and/or translate gene variant nomenclature to increase supervision coverage?

**Comment by ajratner** [comment]
See #89 for more detail...

**Comment by amwenger** [comment]
> a lot of our GV matches are not in HGVS nomenclature as above.

@ajratner What do you mean?  All five examples from #77 are written in HGVS gene-based nomenclature.  There are potentially two caveats:
1. the HGVS variant mention is often in a table
2. the reference gene/isoform is often clear from the context of the paper/table but not explicitly listed with the variant

>  Is there some simple(ish) set of ways to translate or somehow otherwise connect non-HGVS variant mentions to the clinvar data? The clinvar data also has other formats e.g. rs#, AlleleId, etc.

The challenge here is that publications list variants in many different conventions: HGVS, hg18 genome coordinates, hg19 genome coordinates, or even ad hoc conventions.  Ideally, we would extract variants in any convention and convert to a standard.  In my opinion the best standard is the **reference** genome assembly. Variant expressed in assembly coordinates are unambiguous and easily comparable.

It's not too difficult to handle 90% of inter-representation mappings, but getting everything correct is tricky.  I wrote a "90% right" HGVS to genome coordinates converter for my UCSC genome browser.  Try searching "GJA1:c.716G>A" at http://dev-awenger.stanford.edu.

Basically I think the right way to do it is to map all different representations to genome coordinates and then compare.  The genome coordinate mappers will need to be custom-built for each representation.  But I'm not sure that is your main issue here.

@ajratner Could you give some examples of variants that you identify that are not in HGVS nomenclature?

@ajratner How many variant extractions do you have in your training set (independent of phenotype)?
